### DISCUSSION TITLE
跨年要去哪裡呢

### DISCUSSION THREAD

**Discussion by seastar0687** [1101-GS4719-C/Python-Community-GS4719-C#3694] [General]
大家跨年都去哪裡呢
中央因為是18周，所以很多人跨年都還是待在宿舍裡讀書
不過也很多人都會還是跑出去跨年
不過中央附近好像沒什麼特別的地方跨年
我是因為要讀書待在中央，不然因該會去101跨年
大家會特別為跨年跑去哪裡嗎?

**Comment by hollowha** [comment]
可以去中原看煙火，看他們學費噴發的景色
